### PR TITLE
Add sections to "Common Questions"

### PR DESCRIPTION
**Pull request by JSneak**
This is a rough draft of the "What if a member of my club does something inappropriate" section.

### CODE REVIEW

File: clubs/README.md [hunk 1]
```diff
@@ -638,3 +639,9 @@ Teachers/club sponsors/school administrators will sometimes try to ban a club fr
 > When we call ourselves "Hack Club", it means we're building apps, websites, games, and anything else you can imagine.
 
 If that still doesn't work, just reach out to us on [Slack](https://hackclub.slack.com/messages/C0J7C6YET/). A bunch of club leaders have had to deal with this and we're happy to help out.
+
+### What if a member of my club does something inappropriate?
+
+You should warn and ask them stop. If they do not stop, we will remove them from our community spaces temporarily. Repeated, willful breaches of the Code of Conduct will result in a permanent ban.
```
Comment (JSneak) [inline clubs/README.md]: Sorry about that, I copied this over from the conduct page. I'll change it up.

**Comment by JSneak** [comment]
I'm not sure myself what the proper course of action would be. I think that talking to the club's teacher and having the administration deal with it would be best.

**Comment by JSneak** [comment]
Working on adding #985 to the Common Questions section.

File: clubs/README.md [hunk 2]
```diff
@@ -39,6 +39,9 @@ _A step-by-step guide by the Hack Club community._
     - [Reminder for Next Meeting](#reminder-for-next-meeting)
   - [Common Questions](#common-questions)
     - [What if I can’t call my club Hack Club?](#what-if-i-cant-call-my-club-hack-club)
+    - [What if a member of my club does something inappropriate?](#what-if-a-member-of-my-club-does-something-inappropriate)
```
Comment (MaxWofford) [inline clubs/README.md]: Capitalize these

File: clubs/README.md [hunk 3]
```diff
@@ -638,3 +641,17 @@ Teachers/club sponsors/school administrators will sometimes try to ban a club fr
...
+### What if a member of my club does something inappropriate?
+
+You should warn and ask them stop. You'll want to make sure to tell them what is inappropriate about their actions, as they themselves might not realize that are doing it.
+
+If this persists past the warning, then talk to your teacher/club sponsor/school administrator about the proper course of action to take.
+
+Refer to our [conducts page](https://github.com/hackclub/hackclub/blob/master/CONDUCT.md) for more information. 
+
+### What if our adviser does not want to let us meet every week?
+
```
Comment (MaxWofford) [inline clubs/README.md]: Ping me once this is filled in.

Comment (MaxWofford) [inline clubs/README.md]: Any updates? :-)

Comment (JSneak) [inline clubs/README.md]: I pinged you 16 days ago my dude.

File: clubs/README.md [hunk 4]
```diff
@@ -638,3 +641,21 @@ Teachers/club sponsors/school administrators will sometimes try to ban a club fr
...
 
 If that still doesn't work, just reach out to us on [Slack](https://hackclub.slack.com/messages/C0J7C6YET/). A bunch of club leaders have had to deal with this and we're happy to help out.
+
+### What if a member of my club does something inappropriate?
+
+You should warn and ask them stop. You'll want to make sure to tell them what is inappropriate about their actions, as they themselves might not realize that are doing it.
+
+If this persists past the warning, then talk to your teacher/club sponsor/school administrator about the proper course of action to take.
+
+Refer to our [conducts page](https://github.com/hackclub/hackclub/blob/master/CONDUCT.md) for more information.
```
Comment (MaxWofford) [inline clubs/README.md]: "refer to our Code of Conduct"

File: clubs/README.md [hunk 5]
```diff
@@ -638,3 +641,21 @@ Teachers/club sponsors/school administrators will sometimes try to ban a club fr
 > When we call ourselves "Hack Club", it means we're building apps, websites, games, and anything else you can imagine.
 
 If that still doesn't work, just reach out to us on [Slack](https://hackclub.slack.com/messages/C0J7C6YET/). A bunch of club leaders have had to deal with this and we're happy to help out.
+
+### What if a member of my club does something inappropriate?
+
+You should warn and ask them stop. You'll want to make sure to tell them what is inappropriate about their actions, as they themselves might not realize that are doing it.
```
Comment (MaxWofford) [inline clubs/README.md]: " not realize that they are doing something inappropriate"

File: clubs/README.md [hunk 6]
```diff
@@ -638,3 +641,21 @@ Teachers/club sponsors/school administrators will sometimes try to ban a club fr
...
+
+Refer to our [conducts page](https://github.com/hackclub/hackclub/blob/master/CONDUCT.md) for more information. 
+
+### What if our adviser does not want to let us meet every week?
+
+Sometimes, an advisor might not have the time in their schedule to meet weekly. Ask if you can get a second advisor to meet on the weeks that your current one can't.
+
+Alternatively, you could find a public space to meet in and have your club meeting there.
+
+### How do I transition leadership roles to another person?
```
Comment (MaxWofford) [inline clubs/README.md]: I'm partial to dropping this from the FAQ. I don't think many club leaders think much about how to transition their club leadership?

File: clubs/README.md (hunk 4 above)
Comment (JSneak) [inline clubs/README.md]: Done

File: clubs/README.md (hunk 5 above)
Comment (JSneak) [inline clubs/README.md]: Done

**Pull request merged**